lib/utils/error-message.js

```diff
@@ -8,11 +8,9 @@ const messageText = msg => msg.map(line => line.slice(1).join(' ')).join('\n')
 const jsonError = (er, npm, { summary, detail }) => {
   if (npm?.config.loaded && npm.config.get('json')) {
     return {
-      error: {
-        code: er.code,
-        summary: messageText(summary),
-        detail: messageText(detail),
-      },
+      code: er.code,
```

Comment:
This is wrapped in an `error` attribute by the display layer now:

```
        this.#writeOutput(
          output.KEYS.standard,
          meta,
          JSON.stringify({ ...json, error: meta.jsonError }, null, 2)
        )
```

So this change is ok.